requirements.txt

```diff
@@ -1,8 +1,9 @@
 lxml
 bleach
+django==1.11.25
 oauthlib
 mako
 lazy
 git+https://github.com/edx/XBlock.git#egg=XBlock
-git+https://github.com/edx/xblock-utils.git@v1.0.0#egg=xblock-utils==v1.0.0
+git+https://github.com/edx/xblock-utils.git#egg=xblock-utils
```

Comment:
If you're not going to be setting a version, this should be prefixed with a -e for safety.  Otherwise it might get updated correctly.